HW02_A_Graph-Fails.Rmd

```diff
@@ -1,87 +1,95 @@
-  theme(legend.direction = "horizontal") + 
-  theme(legend.position = c(5, 40))
-
-```
-
-### Graph Fail 4
-I wanted just one smoothing line. Just one line, to show the general relationship here. But that's not happening. Instead I'm getting 3 lines, why and fix it please?
-
-```{r}
-mpg %>% 
-ggplot(mapping = aes(x = displ, y = hwy, color = drv)) + 
-  geom_point() + 
-  geom_smooth(se = F) #se = F makes it so it won't show the error in the line of fit
-```
-
-### Graph Fail 5
-I got tired of the points, so I went to boxplots instead. However, I wanted the boxes to be all one color, but setting the color aesthetic just changed the outline? How can I make the box one color, not just the outline? 
-
-Also, the x-axis labels were overlaping, so I rotated them. But now they overlap the bottom of the graph. How can I fix this so axis labels aren't on the graph?
-
-
-```{r}
-ggplot(data = mpg, mapping = aes(x = manufacturer, y = cty, color = manufacturer)) + 
-  geom_boxplot() + 
-  theme(axis.text.x = element_text(angle = 45))
-```
-
-
-
+---
+title: "What went wrong?"
+author: "Robert Gruener"
+date due: "7/13/2020"
+output: github_document
+---
+
+```{r setup, include=TRUE}
+knitr::opts_chunk$set(echo = TRUE, error = TRUE)
+```
+
+## HW02 Part A
+
+### First load packages
+
+```{r load libraries}
+library("ggplot2")
+library("magrittr") #so I can do some piping
+```
+
+### Graph 1
+
+```{r}
+data(mpg) #this is a dataset from the ggplot2 package
+
+mpg %>% 
+  ggplot(mapping = aes(x = city, y = hwy, color = "blue")) %>% 
+  geom_point()
+
+```
+```{r}
+data(mpg) #this is a dataset from the ggplot2 package
+
+ggplot(mpg, mapping = aes(x = cty, y = hwy, color = "blue")) + 
+  geom_point()
+```
+
+### Graph 2
+
+```{r}
+data("mpg")
+ggplot(data = mpg) + 
+  geom_point(mapping = aes(x = displ, y = hwy), color = "blue")
+
+```
+
+
+
+### Graph 3
+
+```{r}
+# graph without legend title
+data("mpg")
+ggplot(mpg, aes(displ, hwy, color = class)) + 
+  # alpha designates opacity
+  # size set to 2
+  geom_point(alpha = 2, size = 2) +
+  theme(legend.direction = "horizontal") + 
+  theme(legend.position = c(0.6, 0.87)) +
+  # remove legend title
+  theme(legend.title = element_blank())
+
+```
+```{r}
+# graph without legend
+data("mpg")
+ggplot(mpg, aes(displ, hwy, color = class)) + 
+  # alpha designates opacity
+  # size set to 2
+  geom_point(alpha = 2, size = 2) +
+  # remove legend
+  theme(legend.position = "none")
+
+```
+
+### Graph 4
+
+```{r}
+mpg %>% 
+ggplot(mapping = aes(x = displ, y = hwy)) + 
+  # moving color to geom_point so that geom_smooth doesn't include it
+  geom_point(aes(color = drv)) + 
+  geom_smooth(se = F) #se = F makes it so it won't show the error in the line of fit
+```
+### Graph 5
+
+```{r}
+ggplot(data = mpg, mapping = aes(x = manufacturer, y = cty, color = manufacturer)) + 
+  # specify boxplot color
+  geom_boxplot(color = "black") + 
+  # add a margin to axis labels
```

Comment:
In my code, I had color and fill = manufacturer within the ggplot function. For your edit, I tried another way: your ggplot function the same except removing the color. and then in geom_boxplot(aes(color = manufacturer , fill = manufacturer)) .. I was able to produce the same plot. nice work!!